src/formatOnSave/shouldFormatOnSave.js

```diff
@@ -55,7 +57,7 @@ const shouldFormatOnSave: (editor: TextEditor) => boolean = _.overEvery([
     isFilePathWhitelisted,
     _.overEvery([noWhitelistGlobsPresent, filePathDoesNotMatchBlacklistGlobs]),
   ]),
-  isFilePathNotEslintignored,
+  shouldNotEslintIgnore,
```

Comment:
This name doesn't seem right to me? We still format even when we should EslintIgnore (as long as the file doesn't match the eslintignore glob). But this name makes it sound like we are only checking the config option.

What about just inlining the `shouldNotEslintIgnore` function? We do this for the disabling checks below.

Reply:
I see what you mean.  My intent was to say that we should not ignore it due to eslint.

Reply:
Hmm, it seems like the good name for it is already taken: `isFilePathEslintIgnored`. Perhaps we should rename the existing one to something like `doesFilePathMatchEslintIgnore` and then we can use that name?